src/main/java/br/com/moip/request/FundingInstrumentRequest.java

```diff
@@ -16,21 +17,27 @@ public FundingInstrumentRequest creditCard(final CreditCardRequest creditCard) {
         return this;
     }
 
+    public FundingInstrumentRequest debitCard(final DebitCardRequest debitCard) {
+        this.debitCard = debitCard;
+        this.method = Method.DEBIT_CARD;
+        return this;
+    }
+
```

Comment:
acho que não precisamos disso. Ainda não vamos aceitar cartões de débito fora do fluxo MPOS

Reply:
Eu criei esse método pra desacoplar na criação do *mposDebitCard* já que é um padrão que é usado no *mposCreditCard* mas entendo que ele está publico e pode ser usado por quem for consumir esse SDK acho que nesse caso podemos deixar ele como **private** e se posteriormente houver necessidade alterá-lo para **public**.

Reply:
entendi! então é melhor mudar ele para private